package.json

```diff
@@ -76,7 +76,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
+      "pre-commit": "sh hooks/pre-commit && lint-staged",
```

Comment:
lint-staged should come before the pre-commit hook.

Reply:
Actually the pre-commit hook should come first, so the lint-staged doesn't even run in a place where it shouldn't.